src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java

```diff
@@ -1,91 +1,84 @@
+        File d = new File(System.getProperty("user.dir") + "/src/test/resources/net/sf/jabref/importer/fileformat");
+        for (File f : d.listFiles()) {
+            files.add(f.getName());
+        }
+        return files;
 
-            Assert.assertFalse(importer.isRecognizedFormat(stream4));
+    }
 
-            Assert.assertFalse(importer.isRecognizedFormat(stream5));
+    @BeforeClass
+    public static void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance();
+    }
 
-            Assert.assertFalse(importer.isRecognizedFormat(stream6));
-        }
+    @Test(expected = IOException.class)
+    public void testImportEntriesException() throws IOException {
+        CopacImporter importer = new CopacImporter();
+        importer.importEntries(null, new OutputPrinterToNull());
     }
 
     @Test
-    public void testImportEntries() throws IOException {
-        Globals.prefs.put("defaultEncoding", StandardCharsets.UTF_8.name());
-
+    public void testGetFormatName() {
         CopacImporter importer = new CopacImporter();
+        Assert.assertEquals("Copac", importer.getFormatName());
+    }
 
-        try (InputStream stream = CopacImporterTest.class
-                .getResourceAsStream("CopacImporterTest1.txt")) {
-            List<BibEntry> entries = importer.importEntries(stream, new OutputPrinterToNull());
-            Assert.assertEquals(1, entries.size());
-            BibEntry entry = entries.get(0);
-
-            Assert.assertEquals("The SIS project : software reuse with a natural language approach", entry.getField("title"));
-            Assert.assertEquals(
-"Prechelt, Lutz and Universität Karlsruhe. Fakultät für Informatik",
-                    entry.getField("author"));
-            Assert.assertEquals("Interner Bericht ; Nr.2/92", entry.getField("series"));
-            Assert.assertEquals("1992", entry.getField("year"));
-            Assert.assertEquals("Karlsruhe :  Universitat Karlsruhe, Fakultat fur Informatik", entry.getField("publisher"));
-            Assert.assertEquals("book", entry.getType());
-        }
+    @Test
+    public void testGetCLIId() {
+        CopacImporter importer = new CopacImporter();
+        Assert.assertEquals("cpc", importer.getCLIId());
     }
 
     @Test
-    public void testImportEntries2() throws IOException {
+    public void testIsRecognizedFormatReject() throws IOException {
         CopacImporter importer = new CopacImporter();
 
-        try (InputStream stream = CopacImporterTest.class
-                .getResourceAsStream("CopacImporterTest2.txt")) {
-            List<BibEntry> entries = importer.importEntries(stream, new OutputPrinterToNull());
-            Assert.assertEquals(2, entries.size());
-            BibEntry one = entries.get(0);
+        List<String> list = testFiles.stream().filter(n -> !n.startsWith("CopacImporterTest"))
+                .filter(n -> !n.startsWith("InspecSilverPlatterImporterTest")).collect(Collectors.toList());
```

Comment:
After talking with @tschechlovdev we both agreed to delete the InspecSilverPlatterImporterTest.txt and replace it with another Inspec file which is not recognized by CopacImporter